### ISSUE TITLE
TEAM_THANOS -INTELLIGENT_LIP_READING_SYSTEM-MACHINE_LEARNING

### ISSUE THREAD

**Issue by somnath-swain** [skillenza-com/MishMash-India-2020#74]
**Before you start, please follow this format for your issue title**:
TEAM_THANOS -INTELLIGENT_LIP_READING_SYSTEM-MACHINE_LEARNING

## ℹ️ Project information

 1. Project Theme- Machine Learning

2. **Project Name:** Intelligent Lip Reading System

3. **Short Project Description:** Lip reading system for the mute persons to interact with AI in automobiles

4. **Team Name:** Team_Thanos

5. **Team Members:** Somnath Swain- @codysom
                                     Prayag Raj Mishra

6. **Demo Link:** https://youtu.be/iV_lIlTJXss

7. **Repository Link(s):** https://github.com/codysom/INTELLIGENT_LIP_READING_SYSTEM

8. **Presentation Link:** https://drive.google.com/file/d/1Jr4ptCIQXivV8Uk-CzXEaNg-lymLEx_3/view?usp=sharing

11. **Azure Services Used-**Didn't use azure service, but can later be integrated for system processing.

## 🔥 Your Pitch
We are nearing towards total automation in the near future but this is futile if it is not accessible by every person. 
Here we have proposed a network system that can help mute persons access the driverless car system using the lips as the communication medium. Our system would basically work on deep neural network for the reconstruction of intelligent speech. This system would use python using tensorflow, SCIPY, CV2 and some other libraries.  It will be using a method for generation of the corresponding sound from the lips. This will result in a more realistic sounding reconstructed speech. This system would consist of an auto-encoder to extract bottlenecks, properties from the auditory spectrogram which will be used as target to the main lip reading network. This main lip reading network would comprise of CNN, LSTM and total connected layers. Here the auto-encoder would be able to reconstruct the original auditory spectrogram with a 90% correlation between real voices. This would also help in improving the quality of reconstructed speech from the main lips reading network.  This system when trained jointly on different speaker will be able to extract Individual speaker characteristics and would hence give promising results by reconstructing intelligible speech with high recognition accuracy. For further integration of LED’s and alarms, we will be using microcontrollers like ARDUINO. This system would ensure the usability of innovation for every human being.

## 🔦 Any other specific thing you want to highlight?
This system can be a boon for the mute persons and also this would make the upcoming automobile projects compatible for all sectors of people

## ✅ Checklist

**Before you post the issue**:
- [x] You have followed the issue title format.
- [x] You have mentioned the correct labels.
- [x] You have provided all the information correctly.
